.travis.yml

```diff
@@ -9,12 +9,20 @@ cache: pip
 python:
 - 3.7
 
+os: linux
```

Comment:
what does this line mean?

Reply:
That the build machine will be a linux machine. 
https://docs.travis-ci.com/user/multi-os/
In future, I'd like to add Windows as well, but TBH the UTs should be OS agnostic anyway...